ingest/src/test/scala/geotrellis/osme/ingest/HttpSlippyTileReaderTest.scala

```diff
@@ -0,0 +1,14 @@
+package geotrellis.osme.ingest
+
+import org.scalatest._
+import geotrellis.spark.SpatialKey
+
+class HttpSlippyTileReaderTest extends FunSuite {
+	test("Print the reading result for (z=1, x=1, y=1)") {
+        val reader = new HttpSlippyTileReader[String]("http://tile.openstreetmap.us/vectiles-highroad/{z}/{x}/{y}.mvt")(fromBytes)
+        println(reader.read(1, 1, 1))
```

Comment:
Test should have asserts, there is not a whole lot you can test here, so you should at least assert that the array is greater than 0 elements long.